src/pages/App.tsx

```diff
@@ -203,6 +203,15 @@ export default function App() {
                     <Route path={`${APP_PATHS.SWAP}/:network/:fromCurrency-to-:toCurrency`} element={<SwapV2 />} />
                     <Route path={`${APP_PATHS.SWAP}/:network/:fromCurrency`} element={<SwapV2 />} />
                     <Route path={`${APP_PATHS.SWAP}/:network`} element={<SwapV2 />} />
+
+                    {SUPPORT_LIMIT_ORDER && (
+                      <>
+                        <Route path={`${APP_PATHS.LIMIT}/:network/:fromCurrency-to-:toCurrency`} element={<SwapV2 />} />
```

Comment:
Does it actually work? `:fromCurrency-to-:toCurrency`

Reply:
yes, swap page already have it 
<img width="756" alt="image" src="https://user-images.githubusercontent.com/33005392/206622777-e60a5384-5168-47cf-bdd8-1be0704f838c.png">